src/smart-switch.js

```diff
@@ -85,21 +85,24 @@ class SmartSwitch extends BasePlugin {
     const responseTimeoutHandler = () => {
       this.logger.warn(`Timeout reached ${this.config.responseTimeoutSec} seconds, loading original source`);
     };
+
+    let url = SmartSwitchConfig['CDN_BALANCER_API_ENDPOINT']
+      .replace('{accountCode}', this.config.accountCode)
+      .replace('{application}', this.config.application || 'default');
```

Comment:
add it to default config instead https://github.com/kaltura/playkit-js-smartswitch/blob/a0c7b07143ce21d9568b6deac6bdf9fa405e54b7/src/smart-switch.js#L9